### DISCUSSION TITLE
Allow disabling openapi/swagger-ui

### DISCUSSION THREAD

**Discussion by tiny-pangolin** [grafana/grafana#67578] [General]
Please allow for the openapi/swagger-ui endpoints to be disabled. using the grafana.ini config file or using environment variables [like gitea](https://docs.gitea.io/en-us/development/api-usage/). I am aware that that users cannot make requests without authentication but I would like to hide this from unauthenticated users with complicated basic auth rules on a reverse proxy.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82749. Feel free to continue the discussion around this there. Thank you!
